hs_core/tests/api/native/test_publish_resource.py

```diff
@@ -25,6 +25,28 @@
 
 
 class TestPublishResource(MockS3TestCaseMixin, TestCase):
+    def test_aaa_settings_are_loaded(self):
```

Comment:
one other issue that needs to be addressed before I can approve:

You have included only this test of settings.

You need to include other tests for the actual deposit.

For example, please replicate these deposit tests:
https://github.com/hydroshare/hydroshare/blob/3.9.0/hs_core/tests/api/native/test_publish_resource.py#L339-L345

But update them so that they are specific to DataCite